backend/social_tasks/views.py

```
@@ -0,0 +1,207 @@
+                        status=status.HTTP_410_GONE,
+                    )
+
+                completion = SocialTaskCompletion.objects.create(
+                    user=user,
+                    task=task,
+                    points_awarded=task.points,
+                    verification_type=task.verification_type,
+                    verification_data=result.audit,
+                )
+        except Exception as exc:
+            logger.exception('Failed to record social task completion: %s', exc)
```

Ruff (0.15.15)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)